### PR TITLE
Add missing monsters

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Adds all missing monsters from the following updates:
[10.90](https://www.tibiawiki.com.br/wiki/Updates/10.90)
[10.80](https://www.tibiawiki.com.br/wiki/Updates/10.80)
[10.70](https://www.tibiawiki.com.br/wiki/Updates/10.70)
[10.50](https://www.tibiawiki.com.br/wiki/Updates/10.50)
[8.54](https://www.tibiawiki.com.br/wiki/Updates/8.54)
[8.5](https://www.tibiawiki.com.br/wiki/Updates/8.5)

[cont]: https://github.com/EPuncker/orts2/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: data/monster/monsters/mooh'tah_master.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Mooh'tah Master" nameDescription="a Mooh'Tah master" race="blood" experience="0" speed="250">
```
Comment (EPuncker) [inline data/monster/monsters/mooh'tah_master.xml:2]:
```suggestion
<monster name="Mooh'Tah Master" nameDescription="a Mooh'Tah master" race="blood" experience="0" speed="250">
```

File: data/monster/monsters/omnivora.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Omnivora" nameDescription="an omnivora" race="blood" experience="750" speed="210">
```
Comment (EPuncker) [inline data/monster/monsters/omnivora.xml:2]:
```suggestion
<monster name="Omnivora" nameDescription="an omnivora" race="venom" experience="750" speed="210">
```

File: data/monster/monsters/omnivora.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		<immunity drunk="1" />
+	</immunities>
+	<elements>
+		<element energyPercent="10" />
+		<element earthPercent="100" />
+		<element firePercent="-10" />
+		<element icePercent="20" />
+		<element deathPercent="20" />
+	</elements>
+	<attacks>
```
Comment (EPuncker) [inline data/monster/monsters/omnivora.xml:51]: remove esse bloco, ja tem attacks la em cima

File: data/monster/monsters/oodok_witchmaster.xml [hunk 4]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Oodok Witchmaster" nameDescription="Oodok Witchmaster" race="blood" experience="550" speed="150">
+	<health now="450" max="450" />
+	<look type="214" corpse="6055" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/oodok_witchmaster.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/omnivora.xml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Omnivora" nameDescription="an omnivora" race="blood" experience="750" speed="210">
+	<health now="1200" max="1200" />
+	<look type="717" corpse="24651" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/omnivora.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/owin.xml [hunk 6]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Owin" nameDescription="Owin" race="blood" experience="5000" speed="300">
+	<health now="9600" max="9600" />
+	<look type="721" corpse="24722" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/owin.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/professor_maxxen.xml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+			<attribute key="areaEffect" value="blueshimmer" />
+		</defense>
+	</defenses>
+	<immunities>
+		<immunity paralyze="1" />
+		<immunity invisible="1" />
+		<immunity drunk="1" />
+	</immunities>
+	<loot>
+		<item id="23574" name="glooth bag" chance="100000" countmax="2" />
```
Comment (EPuncker) [inline data/monster/monsters/professor_maxxen.xml:47]: precisa arrumar o loot

File: data/monster/monsters/rage_of_mazoran.xml [hunk 8]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Rage of Mazoran" nameDescription="a rage of mazoran" race="fire" experience="300" speed="330">
```
Comment (EPuncker) [inline data/monster/monsters/rage_of_mazoran.xml:2]:
```suggestion
<monster name="Rage of Mazoran" nameDescription="Rage of Mazoran" race="fire" experience="300" speed="330">
```

File: data/monster/monsters/rage_of_mazoran.xml [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+	<look type="243" corpse="6324" />
+	<targetchange interval="2000" chance="5" />
+	<flags>
+		<flag pushable="0" />
+		<flag canpushitems="0" />
+		<flag canpushcreatures="0" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
```
Comment (EPuncker) [inline data/monster/monsters/rage_of_mazoran.xml:13]:
```suggestion
		<flag canwalkonfire="1" />
```

File: data/monster/monsters/ragiaz.xml [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-1400" max="-2300" />
```
Comment (EPuncker) [inline data/monster/monsters/ragiaz.xml:18]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-2300" />
```

File: data/monster/monsters/razzagorn.xml [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-1000" max="-2000" />
```
Comment (EPuncker) [inline data/monster/monsters/razzagorn.xml:18]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-2000" />
```

File: data/monster/monsters/renegade_quara_hydromancer.xml [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+		<item id="5910" name="green piece of cloth" chance="3021" />
+		<item id="5914" name="yellow piece of cloth" chance="2915" />
+		<item id="18415" name="green crystal shard" chance="2597" />
+		<item id="2168" name="life ring" chance="1961" />
+		<item id="5895" name="fish fin" chance="1272" />
+		<item id="7632" name="giant shimmering pearl (green)" chance="1219" />
+		<item id="2189" name="wand of cosmic energy" chance="1113" />
+		<item id="2476" name="knight armor" chance="689" />
+		<item id="2155" name="green gem" chance="530" />
+		<item name="giant shimmering pearl" chance="53" />
```
Comment (EPuncker) [inline data/monster/monsters/renegade_quara_hydromancer.xml:59]: falta id

File: data/monster/monsters/renegade_quara_predator.xml [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="1" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="0" max="-465" />
+	</attacks>
+	<defenses armor="40" defense="40">
+		<defense name="speed" interval="2000" chance="15" minspeedchange="300" maxspeedchange="300" duration="7000">
```
Comment (EPuncker) [inline data/monster/monsters/renegade_quara_predator.xml:22]:
```suggestion
		<defense name="speed" interval="2000" chance="15" speedchange="300" duration="7000">
```

File: data/monster/monsters/seacrest_serpent.xml [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+			<attribute key="areaEffect" value="redshimmer" />
+		</attack>
+		<attack name="ice" interval="2000" chance="30" length="8" spread="3" min="0" max="-284">
+			<attribute key="areaEffect" value="watersplash" />
+		</attack>
+		<attack name="physical" interval="2000" chance="10" length="4" spread="0" min="-200" max="-300">
+			<attribute key="areaEffect" value="blueshimmer" />
+		</attack>
+	</attacks>
+	<defenses armor="22" defense="31">
```
Comment (EPuncker) [inline data/monster/monsters/seacrest_serpent.xml:33]:
```suggestion
	<defenses armor="51" defense="51">
```

File: data/monster/monsters/shard_of_corruption.xml [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+		<flag canpushitems="0" />
+		<flag canpushcreatures="0" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" skill="45" min="0" max="-150" />
```
Comment (EPuncker) [inline data/monster/monsters/shard_of_corruption.xml:18]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-150" />
```

File: data/monster/monsters/shulgrax.xml [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-1500" max="-2500" />
```
Comment (EPuncker) [inline data/monster/monsters/shulgrax.xml:18]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-2500" />
```

File: data/monster/monsters/tainted_soul.xml [hunk 17]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<monster name="Tainted Soul" nameDescription="a tainted soul" race="undead" experience="0" speed="170">
+	<health now="250" max="250" />
+	<look type="712" corpse="2843" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/tainted_soul.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/tainted_soul.xml [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+		<flag ignorespawnblock="0" />
+		<flag pushable="0" />
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
```
Comment (EPuncker) [inline data/monster/monsters/tainted_soul.xml:22]: faltando attacks e summon

File: data/monster/monsters/tarbaz.xml [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-1000" max="-2000" />
```
Comment (EPuncker) [inline data/monster/monsters/tarbaz.xml:18]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-2000" />
```

File: data/monster/monsters/tentacle_of_the_deep_terror.xml [hunk 20]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Tentacle Of The Deep Terror" nameDescription="a tentacle of the deep terror" race="venom" experience="0" speed="350">
+	<health now="25500" max="25500" />
+	<look type="681" corpse="0" />
```
Comment (EPuncker) [inline data/monster/monsters/tentacle_of_the_deep_terror.xml:4]: precisa de corpse ja q ele dropa loot

File: data/monster/monsters/the_flaming_orchid.xml [hunk 21]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="The Flaming Orchid" nameDescription="The Flaming Orchid" experience="8500" speed="0">
+	<health now="4000" max="4000" />
+	<look type="150" head="114" body="94" legs="78" feet="79" addons="1" corpse="0" />
```
Comment (EPuncker) [inline data/monster/monsters/the_flaming_orchid.xml:4]: precisa de corpse ja q ele dropa loot

File: data/monster/monsters/the_flaming_orchid.xml [hunk 22]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="The Flaming Orchid" nameDescription="The Flaming Orchid" experience="8500" speed="0">
```
Comment (EPuncker) [inline data/monster/monsters/the_flaming_orchid.xml:2]:
```suggestion
<monster name="The Flaming Orchid" nameDescription="The Flaming Orchid" experience="8500" speed="500">
```

File: data/monster/monsters/the_shatterer.xml [hunk 23]
```diff
@@ -0,0 +1,59 @@
...
+		<flag canpushitems="0" />
+		<flag canpushcreatures="0" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="-200" max="-3000" />
```
Comment (EPuncker) [inline data/monster/monsters/the_shatterer.xml:18]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-3000" />
```

File: data/monster/monsters/true_midnight_asura.xml [hunk 24]
```diff
@@ -0,0 +1,98 @@
...
+	<elements>
+		<element energyPercent="-10" />
+		<element earthPercent="-10" />
+		<element firePercent="10" />
+		<element icePercent="10" />
+		<element holyPercent="30" />
+		<element deathPercent="100" />
+	</elements>
+	<attacks>
+		<attack name="melee" interval="2000" chance="100" min="0" max="-450" />
```
Comment (EPuncker) [inline data/monster/monsters/true_midnight_asura.xml:37]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-450" />
```

File: data/monster/monsters/true_midnight_asura.xml [hunk 25]
```diff
@@ -0,0 +1,98 @@
...
+			<attribute key="areaEffect" value="blacksmoke" />
+		</attack>
+		<attack name="speed" interval="2000" chance="15" radius="1" target="1" speedchange="-100" duration="30000">
+			<attribute key="areaEffect" value="redshimmer" />
+		</attack>
+		<attack name="drunk" interval="2000" chance="10" range="3" target="1" radius="4" duration="4000">
+			<attribute key="areaEffect" value="stun" />
+		</attack>
+	</attacks>
+	<defenses armor="75" defense="55">
```
Comment (EPuncker) [inline data/monster/monsters/true_midnight_asura.xml:52]:
```suggestion
	<defenses armor="75" defense="75">
```

File: data/monster/monsters/tzumrah_the_dazzler.xml [hunk 26]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Tzumrah The Dazzler" nameDescription="Tzumrah The Dazzler" experience="1700" speed="0">
```
Comment (EPuncker) [inline data/monster/monsters/tzumrah_the_dazzler.xml:2]: faltando race e speed

File: data/monster/monsters/unstable_tunnel.xml [hunk 27]
```diff
@@ -0,0 +1,28 @@
...
+	<defenses armor="30" defense="30" />
+	<elements>
+		<element holyPercent="65" />
+	</elements>
+	<immunities>
+		<immunity paralyze="1" />
+		<immunity drunk="1" />
+	</immunities>
+	<summons maxSummons="4">
+		<voice name="Drillworm" interval="2000" chance="15" max="1" />
```
Comment (EPuncker) [inline data/monster/monsters/unstable_tunnel.xml:25]: com esses max 1 ele só vai sumonar 2 bixos

File: data/monster/monsters/weakened_glooth_horror.xml [hunk 28]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Weakened Glooth Horror" nameDescription="a weakened glooth horror" experience="245" speed="240">
```
Comment (EPuncker) [inline data/monster/monsters/weakened_glooth_horror.xml:2]: faltando race

File: data/monster/monsters/werebadger.xml [hunk 29]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Werebadger" nameDescription="a werebadger" race="blood" experience="1600" speed="260">
+	<health now="1700" max="1700" />
+	<look type="729" corpse="24723" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/werebadger.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/werebear.xml [hunk 30]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Werebear" nameDescription="a werebear" race="blood" experience="2100" speed="220">
+	<health now="2400" max="2400" />
+	<look type="720" corpse="24666" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/werebear.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/werebear.xml [hunk 31]
```diff
@@ -0,0 +1,78 @@
...
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="1" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="1" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="2000" min="50" max="-485" />
```
Comment (EPuncker) [inline data/monster/monsters/werebear.xml:24]:
```suggestion
		<attack name="melee" interval="2000" min="0" max="-485" />
```

File: data/monster/monsters/zamulosh.xml [hunk 32]
```diff
@@ -0,0 +1,65 @@
...
+		<flag canpushitems="1" />
+		<flag canpushcreatures="1" />
+		<flag staticattack="90" />
+		<flag targetdistance="1" />
+		<flag canwalkonenergy="0" />
+		<flag canwalkonfire="0" />
+		<flag canwalkonpoison="0" />
+	</flags>
+	<attacks>
+		<attack name="melee" interval="3000" min="-1500" max="-2300" />
```
Comment (EPuncker) [inline data/monster/monsters/zamulosh.xml:18]:
```suggestion
		<attack name="melee" interval="3000" min="0" max="-2300" />
```

File: data/monster/monsters/wereboar.xml [hunk 33]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Wereboar" nameDescription="a wereboar" race="blood" experience="2000" speed="300">
+	<health now="2200" max="2200" />
+	<look type="721" corpse="24722" />
+	<targetchange interval="2000" chance="0" />
```
Comment (EPuncker) [inline data/monster/monsters/wereboar.xml:5]:
```suggestion
	<targetchange interval="2000" chance="5" />
```

File: data/monster/monsters/bullwark.xml [hunk 34]
```diff
@@ -30,7 +30,9 @@
 			<attribute key="shootEffect" value="whirlwindclub" />
 			<attribute key="areaEffect" value="explosion" />
 		</attack>
-		<attack name="bullwark paralyze" interval="2000" chance="6" />
+		  <attack name="speed" interval="2000" chance="6" target="0" length="3" spread="2" minspeedchange="-550" maxspeedchange="-700" duration="50000">
+            <attribute key="areaEffect" value="blackspark" />
```
Comment (EPuncker) [inline data/monster/monsters/bullwark.xml]: indentation

File: data/monster/monsters/devourer.xml [hunk 35]
```diff
@@ -25,7 +25,10 @@
 			<attribute key="areaEffect" value="greenbubble" />
 		</attack>
 		<attack name="devourer wave" interval="2000" chance="5" min="-50" max="-150" />
-		<attack name="devourer paralyze" interval="2000" chance="9" />
+		<attack name="speed" interval="2000" chance="9" range="7" radius="4" target="1" minspeedchange="-550" maxspeedchange="-750" duration="15000">
+            <attribute key="shootEffect" value="poison" />
```
Comment (EPuncker) [inline data/monster/monsters/devourer.xml]: indentation

File: data/monster/monsters/enthralled_demon.xml [hunk 36]
```diff
@@ -31,7 +31,10 @@
 		<attack name="firefield" interval="2000" chance="14" range="7" radius="1" target="1">
 			<attribute key="shootEffect" value="fire" />
 		</attack>
-		<attack name="demon paralyze" interval="2000" chance="10" range="7" />
+		<attack name="speed" interval="2000" chance="25" range="7" radius="4" target="1" minspeedchange="-300" maxspeedchange="-450" duration="25000">
+            <attribute key="shootEffect" value="suddendeath" />
```
Comment (EPuncker) [inline data/monster/monsters/enthralled_demon.xml]: indentation

File: data/monster/monsters/lisa.xml [hunk 37]
```diff
@@ -31,7 +31,9 @@
 		<attack name="manadrain" interval="2000" chance="13" radius="8" target="0" min="-100" max="-200">
 			<attribute key="areaEffect" value="poison" />
 		</attack>
-		<attack name="lisa paralyze" interval="2000" chance="12" />
+		<attack name="speed" interval="2000" chance="12" range="7" radius="4" target="1" minspeedchange="-200" maxspeedchange="-450" duration="15000">
+            <attribute key="areaEffect" value="blackspark" />
```
Comment (EPuncker) [inline data/monster/monsters/lisa.xml]: indentation

File: data/monster/monsters/professor_maxxen.xml [hunk 38]
```diff
@@ -28,7 +28,10 @@
 		<attack name="war golem skill reducer" interval="2000" chance="22" />
 		<attack name="war golem electrify" interval="2000" range="2" chance="16" />
 		<attack name="glooth fairy skill reducer" interval="2000" chance="20" />
-		<attack name="glooth fairy paralyze" interval="2000" chance="9" />
+		<attack name="speed" interval="2000" chance="9" range="7" radius="4" target="1" minspeedchange="-600" maxspeedchange="-800" duration="20000">
+            <attribute key="shootEffect" value="poison" />
```
Comment (EPuncker) [inline data/monster/monsters/professor_maxxen.xml]: indentation

File: data/monster/monsters/devourer.xml (hunk 35 above)
Comment (omarcopires) [reply data/monster/monsters/devourer.xml]: Formatting is correct

File: data/monster/monsters/professor_maxxen.xml (hunk 38 above)
Comment (omarcopires) [reply data/monster/monsters/professor_maxxen.xml]: Formatting is correct

File: data/monster/monsters/lisa.xml (hunk 37 above)
Comment (omarcopires) [reply data/monster/monsters/lisa.xml]: Formatting is correct

File: data/monster/monsters/enthralled_demon.xml (hunk 36 above)
Comment (omarcopires) [reply data/monster/monsters/enthralled_demon.xml]: Formatting is correct

File: data/monster/lua/orts/demon_summoner.lua [hunk 39]
```diff
@@ -0,0 +1,77 @@
...
+monster.corpse = 0
+monster.speed = 0
+monster.manaCost = 0
+
+monster.changeTarget = {
+	interval = 4000,
+	chance = 10
+}
+
+monster.strategiesTarget = {
```
Comment (EPuncker) [inline data/monster/lua/orts/demon_summoner.lua:22]: remove esse bloco todo

File: data/monster/lua/orts/demon_summoner.lua [hunk 40]
```diff
@@ -0,0 +1,77 @@
...
+	nearest = 100
+}
+
+monster.flags = {
+	attackable = true,
```
Comment (EPuncker) [inline data/monster/lua/orts/demon_summoner.lua:27]:
```suggestion
	attackable = false,
```

File: data/monster/lua/orts/demon_summoner.lua [hunk 41]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+monster.summon = {
+	maxSummons = 1,
+	summons = {
+		{name = "Demon", chance = 100, interval = 1000, count = 1}
+	}
+}
+
+monster.voices = {
```
Comment (EPuncker) [inline data/monster/lua/orts/demon_summoner.lua:47]: remove esse bloco

File: data/monster/lua/orts/demon_summoner.lua [hunk 42]
```diff
@@ -0,0 +1,77 @@
...
+	{type = COMBAT_LIFEDRAIN, percent = 100},
+	{type = COMBAT_DROWNDAMAGE, percent = 100},
+	{type = COMBAT_ICEDAMAGE, percent = 100},
+	{type = COMBAT_HOLYDAMAGE, percent = 100},
+	{type = COMBAT_DEATHDAMAGE, percent = 100}
+}
+
+monster.immunities = {
+	{type = "paralyze", condition = true},
+	{type = "outfit", condition = false},
```
Comment (EPuncker) [inline data/monster/lua/orts/demon_summoner.lua:72]:
```suggestion
	{type = "outfit", condition = true},
```

File: data/monster/lua/orts/demon_summoner.lua [hunk 43]
```diff
@@ -0,0 +1,77 @@
...
+	{type = "invisible", condition = true},
+	{type = "bleed", condition = false}
```
Comment (EPuncker) [inline data/monster/lua/orts/demon_summoner.lua:74]: remove essa linha

File: data/monster/lua/orts/immune_azerus.lua [hunk 44]
```diff
@@ -0,0 +1,94 @@
...
+	{type = COMBAT_MANADRAIN, percent = 100},
+	{type = COMBAT_DROWNDAMAGE, percent = 100},
+	{type = COMBAT_ICEDAMAGE, percent = 100},
+	{type = COMBAT_HOLYDAMAGE , percent = 100},
+	{type = COMBAT_DEATHDAMAGE , percent = 100}
+}
+
+monster.immunities = {
+	{type = "paralyze", condition = true},
+	{type = "outfit", condition = false},
```
Comment (EPuncker) [inline data/monster/lua/orts/immune_azerus.lua:89]: remove esse e o bleed

File: data/monster/monsters/strong_glooth_horror.xml [hunk 45]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Strong Glooth Horror" nameDescription="a strong glooth horror" race="venom" experience="1500" speed="0">
```
Comment (EPuncker) [inline data/monster/monsters/strong_glooth_horror.xml:2]: speed 0 mesmo?

File: data/monster/monsters/werebadger.xml [hunk 46]
```diff
@@ -0,0 +1,83 @@
...
+		<element firePercent="-5" />
+		<element icePercent="-5" />
+		<element holyPercent="-5" />
+		<element deathPercent="40" />
+	</elements>
+	<voices interval="5000" chance="10">
+		<voice sentence="SNUFFLE" />
+		<voice sentence="Sniff Sniff" />
+	</voices>
+	<loot>
```
Comment (EPuncker) [inline data/monster/monsters/werebadger.xml:62]: falta summons

**Review by EPuncker — APPROVED** [review]

**Pull request merged**